### ISSUE TITLE
Problem with Conv1D and Embedding Layer

### ISSUE THREAD

**Issue by protti** [fchollet/keras#6756]
Hi, I'm trying to adapt [this example](https://github.com/fchollet/keras/blob/master/examples/imdb_cnn.py) to my code. I have a set of 60 records, and each one has 4 attributes and the class, they are natural number. I want to create a NN that training about this.

This is the code that I try to wrote:

  

    model = Sequential()
    model.add(Embedding(60,4))
    model.add(Dropout(0.2))
    # we add a Convolution1D, which will learn filters
    # word group filters of size filter_length:
    model.add(Conv1D(25,10,
                     padding='valid',
                     activation='relu',
                     strides=1))
    # we use max pooling:
    model.add(GlobalMaxPooling1D())
    # We add a vanilla hidden layer:
    model.add(Dense(25))
    model.add(Dropout(0.2))
    model.add(Activation('relu'))

    # We project onto a single unit output layer, and squash it with a sigmoid:
    model.add(Dense(1))
    model.add(Activation('sigmoid'))
    model.compile(loss='binary_crossentropy',
                 optimizer='adam',
                 metrics=['accuracy'])
    model.fit(X, Y)

But the compiler says this: https://pastebin.com/9WiHtKfJ

Thanks for the help!
